### ISSUE TITLE
Clean builds don't seem to be building against most recent develop changes

### ISSUE THREAD

**Issue by bmpersc** [trilinos/Trilinos#1940]
@trilinos/framework, @csiefer2 pointed out that the fixes he had made to address the issues #1907 didn't fix the clean builds like he expected. Looking into this the clean builds seem to be building against cf2c2c4b3c16738afef4856f2a587d597be4daba which is 15 commits behind develop. It appears that the clean builds are not updating properly. A quick look through the logs did not show an obvious error.

**Comment by jhux2** [comment]
@bmpersc Specialized builds don't appear to be updating either, e.g.,
https://testing.sandia.gov/cdash/buildSummary.php?buildid=3195972

:worried:

**Comment by csiefer2** [comment]
🤦‍

**Comment by bmpersc** [comment]
@jhu @csiefer2 I have tracked this down to the nightly mirror of Trilinos didn't update last night so this would have impacted a lot of builds. I was able to update the mirror today so it isn't a systemic issue. However, looking through the logs I have it seems like the update of the mirror has failed 4 times over the last 3 weeks, each with a very different error. 75% have happened on Wednesday so there is probably something that is happening on that day and time to either degrade network or the system performance. However, I don't know if that is from our end or github's. I'll continue monitoring this, but for now the symptom has at least been resolved.
